cmd/cape/cmd/token.go

```diff
@@ -67,12 +81,24 @@ func token(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	_, err = cmd.OutOrStdout().Write([]byte(tokenString + "\n"))
-	if err != nil {
-		return err
+	if o != "json" {
+		_, err = cmd.OutOrStdout().Write([]byte(tokenString + "\n"))
+		if err != nil {
+			return err
+		}
 	}
 
-	return nil
+	output := struct {
+		ID       string `json:"function_id"`
+		Token    string `json:"function_token"`
+		Checksum string `json:"function_checksum"`
+	}{
+		ID:       functionID,
+		Token:    fmt.Sprintf("%x", tokenString),
```

Comment:
```suggestion
		Token:   tokenString,
```

Can you just print out the tokenString as is?